e2e/smoke_test/smokeTest.spec.js

```diff
@@ -1,29 +1,41 @@
-const { setup, match, Carbs, Bolus, Overrides, Settings, SettingDefault, CGMModel, CGMEffect } = require('../../src/index');
+const { setup, Carbs, Status, Bolus, Overrides, Settings, SettingDefault, CGMModel, CGMEffect } = require('../../src/index');
 
 describe('smoke test', () => {
     var settings;
     var carbs;
     var bolus;
     var overrides;
+    var status;
     beforeAll(async () => {
         await setup.LaunchLoop();
         settings = new Settings();
         carbs = new Carbs();
         bolus = new Bolus();
         overrides = new Overrides();
+        status = new Status();
     });
-    describe('main screen', () => {
+    describe('status screen', () => {
         it('has Active Carbohydrates section', async () => {
-            await expect(match.accessible.Label('Active Carbohydrates')).toExist();
+            await status.OpenActiveCarbohydratesChart();
```

Comment:
[nit] I'm confused, why would `status` open the active carbohydrates chart? Perhaps this is a naming thing?